app/_src/deck/guides/distributed-configuration.md

```diff
@@ -88,7 +88,22 @@ configuration.
 You can delete only a subset of entities sharing a tag using the `--select-tag`
 flag on the `reset` command.
 
-## Initial setup problem
+## Convert distributed configuration into one file
+
+You can convert multiple distributed declarative configuration files into one
```

Comment:
into one Gateway file is more accurate don't you think?

cc @GGabriele

Reply:
Honestly I'd have used `You can convert multiple distributed decK configuration files into one Gateway file`.
The reason for replacing `declarative` with `decK` is that `declarative` is a very specific format in Kong, which is mostly related with the DB-less mode.

That said, I see that `declarative configuration` is consistently used in the docs, so it's better to not diverge from it.

Overall, I think the suggestion from Vincent makes sense.

Reply:
> The reason for replacing declarative with decK is that declarative is a very specific format in Kong, which is mostly related with the DB-less mode.
> That said, I see that declarative configuration is consistently used in the docs, so it's better to not diverge from it.

Not always the best reason to keep doing things a certain way 😄 . Ill use your phrasing.

Curious: what _is_ the difference? Is there any different in format? The terms are generally used interchangeably, as you've noticed, and I've never heard anyone mention a difference - I didn't know they weren't referring to the same thing till now, eek.